pkcs12/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pkcs12"
-version = "0.0.0"
+version = "0.1.0"
```

Comment:
Please don't bump the version. We do separate release PRs for this.

Reply:
done